tests/test_custom_schema_fields.py

```diff
@@ -42,7 +54,18 @@ def foo():
         "name": {
             "title": "Name",
             "type": "string",
-        }
+        },
+        "description": (
+            {
+                "title": "Description",
+                "type": IsList("string", "null", check_order=False),
+            }
+            if PYDANTIC_V2
+            else {
+                "title": "Description",
+                "type": "string",
+            }
+        ),
```

Comment:
```suggestion
        "description": {
            "title": "Description",
            "type": (
                IsList("string", "null", check_order=False)
                if PYDANTIC_V2
                else "string"
            ),
        },
```
I think it's better to simplify it. Only the value of `type` field is different